### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by gatlinnewhouse** [lwindolf/liferea#520]
On 64 bit Arch Linux with `Liferea 1.12-rc3`:

```zsh
deleuze@rhizome $ liferea --debug-verbose

(liferea:15047): Gtk-CRITICAL **: gtk_event_controller_reset: assertion 'GTK_IS_EVENT_CONTROLLER (controller)' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 37, in do_activate
    raise ValueError("Failed to unlock GnomeKeyring, error "+result)
TypeError: must be str, not Result
zsh: segmentation fault (core dumped)  liferea --debug-verbose
```

**Comment by gatlinnewhouse** [comment]
Ran it with `gdb` and got the following error:

```gdb
Thread 1 "liferea" received signal SIGSEGV, Segmentation fault.
0x00007ffff3e51bf9 in gtk_tree_view_get_model () from /usr/lib/libgtk-3.so.0
```

**Comment by lwindolf** [comment]
1.12.1 introduced a fix for this #504. Please retest!
